### ISSUE TITLE
Feature Request - Autocorrect Asterisk

### ISSUE THREAD

**Issue by jwest75674** [googlesamples/gboard-dev-samples#1]
I suspect that this is not the correct repo, but I can't seem to find a better one. Hopefully someone can pass this along:

Autocorrect is a modern marvel, but it's imperfect. 
In my neck of the woods, Typos and autocorrect mistakes are often followed with a correction following as asterisk:
```
We'll, it has been fun!
Well*
```

Sometimes autocorrect strikes twice:
```
We'll it has been fun!
We'll*
We'll**
WELL***
```

Why hasn't anyone added the simple logic of:
`If first word ends with asterisk, undo preceding autocorrections`

I hope this small feature request can find it's way to the appropriate folks, but understand if this issue is closed too!

**Comment by FnBrian79** [comment]
Good point I think that should be adopted and considered at least of all any little improvements are needed wherever they can be put most effectively.
